cmd/haproxy-cfg-generator/util.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2020 Authors of Arktos.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/uuid"
+	"k8s.io/klog"
+)
+
+const ShellToUse = "bash"
+
+type FontColor string
+
+const (
+	ResetColor  = FontColor("\033[0m")
+	RedColor    = FontColor("\033[31m")
+	GreenColor  = FontColor("\033[32m")
+	YellowColor = FontColor("\033[33m")
+	BlueColor   = FontColor("\033[34m")
+	PurpleColor = FontColor("\033[35m")
+	CyanColor   = FontColor("\033[36m")
+	GrayColor   = FontColor("\033[37m")
+	WhiteColor  = FontColor("\033[97m")
+)
+
+func ApplyColor(s string, color FontColor) string {
+	return string(color) + s + string(ResetColor)
+}
+
+func RandomString(length int) string {
+	// If we see a crazy length, set it the default length of 8
+	if length <= 0 || length > 128 {
+		length = 8
```

Comment:
add a log here indicating the cut in case of length > 128

otherwise this silent action might be hard to debug if it is triggered

Reply:
Actually, this function was not in use in this tool. I removed it (and another util func) completely. :-)